### ISSUE TITLE
Syntax Error on Python 3.7

### ISSUE THREAD

**Issue by saulshanabrook** [mbdevpl/typed-astunparse#5]
I am trying to run this on python `3.7.0b3` and am getting a SyntaxError when I import it. Is `async` a reserved keyword now?

```bash
$ python3.7
Python 3.7.0b3 (v3.7.0b3:4e7efa9c6f, Mar 29 2018, 08:11:44)
[Clang 6.0 (clang-600.0.57)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> import typed_ast.ast3
>>> import typed_astunparse
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/typed_astunparse/__init__.py", line 12, in <module>
    from .unparser import Unparser
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/typed_astunparse/unparser.py", line 248
    def _generic_FunctionDef(self, t, async=False):
                                          ^
SyntaxError: invalid syntax
>>>
```

**Comment by mbdevpl** [comment]
Indeed, `async` becomes a keyword since 3.7. This issue was resolved in development version of upstream https://github.com/simonpercivall/astunparse/pull/15 but the fixed version was not yet published, so unless you use git-cloned version of upstream _astunparse_ package, you'll encounter the same error there.

Fixing the `async` in my codebase is very simple - I've already almost done it, only the function signatures remain. And I'll fix them too as soon as the upstream fix is published.

**Comment by saulshanabrook** [comment]
@mbdevpl Sounds good, thank you.

**Comment by simonpercivall** [comment]
@mbdevpl The 1.6.0 version of astunparse fixes this.

**Comment by mbdevpl** [comment]
@simonpercivall Thanks! I'll also make a new release reflecting recent changes.

**Comment by mbdevpl** [comment]
@saulshanabrook Fixed in typed-astunparse 2.0.4
